ui/pages/confirmations/components/confirm/info/shared/transaction-details/transaction-details.tsx

```diff
@@ -53,13 +56,16 @@ export const RecipientRow = ({ recipient }: { recipient?: Hex } = {}) => {
   const t = useI18nContext();
   const { currentConfirmation } = useConfirmContext<TransactionMeta>();
   const to = recipient ?? currentConfirmation?.txParams?.to;
+  const isBatchTransaction =
```

Comment:
Rather than relying on the type, as we may add more batch types in future, should we instead rely on checking the length of `nestedTransactions` in `TransactionMeta`?

As that's what triggers the nested transaction data component, plus the metrics.

Reply:
Perfect, changed to check the length of the `nestedTransactions`.